properties/dapr_nuget.props

```
@@ -1,20 +1,20 @@
 <Project>
   <!-- Nuget package properties when packed using dotnet pack. -->
   <PropertyGroup>
-    <Authors>dapr.io</Authors>
-    <PackageRequireLicenseAcceptance>true</PackageRequireLicenseAcceptance>
+    <Owners>Microsoft,Azure-SDK</Owners>
```

Nit: should we have space between 2 words?
```suggestion
    <Owners>Microsoft, Azure-SDK</Owners>
```